<ui version="4.0" >
 <class>TkrClass</class>
 <widget class="QWidget" name="TkrClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Book Antiqua</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Tkr</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QTreeWidget" name="treeWidget" >
     <column>
      <property name="text" >
       <string>Hall</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>DCServer IP</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>DCS Serial No.</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Certificate</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Manufacturer</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="icon" >
        <iconset resource="cinecastui.qrc" >
         <normaloff>:/CineCastUi/Resources/right.png</normaloff>:/CineCastUi/Resources/right.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="icon" >
        <iconset resource="cinecastui.qrc" >
         <normaloff>:/CineCastUi/Resources/left.png</normaloff>:/CineCastUi/Resources/left.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3" >
       <property name="icon" >
        <iconset resource="cinecastui.qrc" >
         <normaloff>:/CineCastUi/Resources/delete.png</normaloff>:/CineCastUi/Resources/delete.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="minimumSize" >
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Hall</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="minimumSize" >
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>DC Server IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="minimumSize" >
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>DCS Serial No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Manufacturer</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QRadioButton" name="radioButton" >
           <property name="text" >
            <string>Doremi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2" >
           <property name="text" >
            <string>Dolby</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_3" >
           <property name="text" >
            <string>GDC</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <widget class="QLineEdit" name="lineEdit_4" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4" >
          <property name="text" >
           <string>Load Creditficate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="cinecastui.qrc" />
 </resources>
 <connections/>
</ui>
